.github/CODEOWNERS

```diff
@@ -36,23 +36,14 @@
 /client/state/gsuite-users @Automattic/letero
 
 # Payments
-/client/components/credit-card-form-fields/ @Automattic/shilling
-/client/components/payment-logo/ @Automattic/shilling
-/client/lib/cart-values/ @Automattic/shilling
-/client/lib/checkout/ @Automattic/shilling
-/client/lib/siftscience/ @Automattic/shilling
-/client/lib/store-transactions/ @Automattic/shilling
-/client/me/memberships/ @Automattic/shilling
-/client/me/purchases/ @Automattic/shilling
-/client/my-sites/checkout/ @Automattic/shilling
+/client/me/purchases/add-new-payment-method/ @Automattic/shilling
+/client/me/purchases/billing-history/ @Automattic/shilling
+/client/me/purchases/manage-purchase/ @Automattic/shilling
+/client/me/purchases/payment-methods/ @Automattic/shilling
+/client/my-sites/checkout/checkout-main-wrapper.tsx @Automattic/shilling
+/client/my-sites/checkout/src/ @Automattic/shilling
+/client/my-sites/checkout/get-thank-you-page-url/ @Automattic/shilling
 /client/my-sites/purchases/ @Automattic/shilling
-/client/state/billing-transactions/ @Automattic/shilling
-/client/state/memberships/ @Automattic/shilling
```

Comment:
Yes, I thought about that but I also suspect they're the only ones who are likely to be editing those files, and even if they aren't they might not want to be pinged about changes. Some of the things I removed for Shilling, for example, are still code that we work on but I think we only need to be pinged for locations that are likely to be broken by other teams. 

For me, CODEOWNERS is reactive: lines are here because PRs that did not ping us have caused problems for us in the past. Based on that I'm assuming that Gold may want to add themselves only if they start to see problems. Nevertheless, I'll let them know in Slack.

Reply:
Yep, makes sense to me.